pipelines/resources/deploy-ztp-edgeclusters-minimal.yaml

```diff
@@ -0,0 +1,291 @@
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  # Deploy Workers
+  - name: deploy-workers
+    taskRef:
+      name: edgecluster-deploy-workers
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-metallb
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  # Deploy ZTPFWUI
+  - name: deploy-ui
+    taskRef:
+      name: edgecluster-deploy-ui
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-metallb
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  - name: deploy-lso
+    taskRef:
+      name: edgecluster-deploy-lso
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-workers
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  # Deploy ODF
+  - name: deploy-odf
+    taskRef:
+      name: edgecluster-deploy-odf
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-lso
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+  # Deploy Disconnected Registry
+  - name: deploy-disconnected-registry
+    taskRef:
+      name: edgecluster-deploy-disconnected-registry-edgeclusters
+    params:
+      - name: edgeclusters-config
+        value: $(params.edgeclusters-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+      - name: pipeline-name
+        value: $(context.pipelineRun.name)
+    runAfter:
+      - deploy-odf
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+    # Deploy Disconnected Registry Pull secret
+  - name: deploy-disconnected-registry-ps
```

Comment:
Note that this and the next step will make the cluster point to the internal registry that doesn't have images.

Reply:
We could remove it in a subsequent MR - if needed.
The goal is to be able to test most of the tasks. WDYT?